proyecto/src/main/java/com/proyecto/proyecto/Utileria.java

```diff
@@ -22,22 +22,25 @@ public static String guardarArchivo(MultipartFile multiPart, String ruta) {
             return null;
         }
     }
-    */
-    public static boolean saveToFile(String path, String data)
+
+    public static String saveToFile(String path, String data)
     {
         try
         {
             FileOutputStream fileStream = new FileOutputStream(new File(path));
             OutputStreamWriter output = new OutputStreamWriter(fileStream);
             output.write(data);
             output.close();
+            return path;
         }
         catch (Exception e)
         {
-            return false;
+            return null;
```

Comment:
#should siendo que el método devuelve la ruta del parámetro sin alterarla, es mejor que siga devolviendo `boolean` y actuar en consecuencia, sin exponer la probabilidad de modificar innecesariamente la ruta que se suministró.